Packs/RecordedFutureASI/README.md

```diff
@@ -0,0 +1,10 @@
+# Pack Documentation
+This pack is used to access Recorded Future ASI data to pull risk rules into XSOAR as incidents.
+
```

Comment:
### What does this pack do?
#### This pack enables security teams to:

- Access a unified risk management from the most popular SOAR platform.
- Visualize to the most critical risks within your organization
- Identify security incidents filtered by severity (critical, medium and low)
- See the full context of the incident, including CVE id, name, description, and affected hostnames.

Reply:
are you suggesting to put this section from the `RecordedFutureASI_description.md` in the Pack README as well replacing the `# Pack Documentation` section?

Reply:
done